src/character.cpp

```diff
@@ -1512,10 +1554,10 @@ int Character::get_working_arm_count() const
     }
 
     int limb_count = 0;
-    if( !is_limb_disabled( body_part_arm_l ) ) {
+    if( has_limb( body_part_arm_l ) && !is_limb_disabled( body_part_arm_l ) ) {
```

Comment:
Shouldn't this take advantage of the new part type and look for an arm instead of that one specific type of arm?

Reply:
i would like to address this in this pr, and not in the previous pr (which is what that commit is on)